test/rhsmlib_test/test_pkg_arches.py

```diff
@@ -0,0 +1,80 @@
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
```

Comment:
since sub-man is py3-only, directly use `unittest`

Reply:
Implemented as suggested.